indico/modules/events/editing/controllers/backend/timeline.py

```diff
@@ -39,10 +39,26 @@
 
 
 class RHEditingUploadFile(UploadFileMixin, RHContributionEditableBase):
+    EDITABLE_REQUIRED = False
+
     def get_file_context(self):
         return 'event', self.event.id, 'editing', self.contrib.id, self.editable_type.name
 
 
+class RHEditingUploadPaperLastRevision(RHEditingUploadFile):
+    @use_kwargs({
+        'id': fields.Int()
```

Comment:
Is it worth changing this to a list of numbers (ids) instead?

Reply:
when would the be useful?

Reply:
Since `id` is the only parameter used. Instead of `[{ "id": 17 }, ...]`. I would have `[17,...]`.

Reply:
What I mean is you never have more than one `id` argument... so it's `{id: 17}` vs `17`.. and I don't think webargs can process the latter

Reply:
Never mind, I mixed this with `RHSubmitNewRevision`. Forgot only one file was accepted here.